Misc/Lobby/oa_tanabata.csv

```diff
@@ -130,21 +130,21 @@ result53#1,"""ブルーノ"""
 result54#1,"""せーわ"""
 result55#1,"""機械仕掛けのさおリリーパ"""
 result56#1,"""クリきんと"""
-result57#1,"""mashiro@uyo"""
+result57#1,"""Mashiro@uyo"""
```

Comment:
Please keep quiet the tanabata names

Reply:
Done

Reply:
No it is not, there still changes in oa_tanabata.csv

Reply:
ahm.... done?

Reply:
I still see changes in oa_tanabata.csv
https://github.com/Arks-Layer/PSO2ENPatchCSV/pull/818/files#diff-56acfed6086053c83194edf627c997b0

Reply:
and there are STILL CHANGES in oa_tanabata.csv